app/bundles/FormBundle/Views/Form/details.html.php

```diff
@@ -319,15 +319,15 @@
                 <div class="modal-body">
                     <p><?php echo $view['translator']->trans('mautic.form.form.help.automaticcopy'); ?></p>
                     <h3><?php echo $view['translator']->trans('mautic.form.form.help.automaticcopy.js'); ?></h3>
-                    <textarea class="form-control" readonly onclick="this.setSelectionRange(0, this.value.length);">&lt;script type="text/javascript" src="<?php echo $view['router']->url(
+                    <textarea class="form-control" readonly onclick="this.setSelectionRange(0, this.value.length);" aria-label="Form generation">&lt;script type="text/javascript" src="<?php echo $view['router']->url(
```

Comment:
```suggestion
                    <textarea class="form-control" readonly onclick="this.setSelectionRange(0, this.value.length);" aria-label="Generate form">&lt;script type="text/javascript" src="<?php echo $view['router']->url(
```

I think that the aria label should say what the feature actually does using active tense? But open to discussion on this. I feel like if it were being read out to me, 'Generate form' may make more sense than 'Form generation'.